python/example_robot_data/robots_loader.py

```diff
@@ -483,8 +483,16 @@ class DoublePendulumLoader(RobotLoader):
     urdf_subpath = "urdf"
 
 
-def loadDoublePendulum():
+class DoublePendulumContinuousLoader(RobotLoader):
```

Comment:
This class should inherit from `DoublePendulumLoader` and only override its `urdf_filename`

Reply:
Will fix asap